files/help/ares-device.help

```diff
@@ -16,9 +16,15 @@
     },
     "options" : {
         "cmdOpt":"option",
-        "default" : ["system", "session", "device", "device-list", "help", "version", "verbose"],
+        "default" : ["system", "session", "capture", "display", "device", "device-list", "help", "version", "verbose"],
         "system" : "-i, --system-info @TAB@ Display the device system information",
         "session" : "-s, --session-info @TAB@ Display the device session information",
+        "capture" : [
+            "-c, --capture <OUTPUT_PATH> @TAB@ Capture screen and save the file to local",
```

Comment:
Capture screen and save captured file in host PC.
I think this is proper..

Reply:
I replaced it host machine, which is used in ares-pull help.
Done.